packages/web/src/components/tipping/tip-audio/SendTip.tsx

```diff
@@ -163,14 +171,43 @@ export const SendTip = () => {
   )
 
   return receiver ? (
-    <div className={styles.container}>
+    <div
```

Comment:
nit: maybe worth pulling this out into a render fn with a few separate returns, this nested ternary is getting a bit hairy